ee/server/service/software.go

```diff
@@ -16,3 +17,14 @@ func (svc *Service) SoftwareByID(ctx context.Context, id uint, teamID *uint, _ b
 	// reuse SoftwareByID, but include cve scores in premium version
 	return svc.Service.SoftwareByID(ctx, id, teamID, true)
 }
+
+func (svc *Service) ListFleetMaintainedApps(ctx context.Context, teamID uint, opts fleet.ListOptions) ([]fleet.FleetMaintainedAppAvailable, *fleet.PaginationMetadata, error) {
+	svc.authz.SkipAuthorization(ctx)
```

Comment:
missing authorization check here

Reply:
Oh shoot! Nice catch 🙂

Reply:
Do we have an auth path specifically for FMA? Should I reuse the one we use for VPP?

Reply:
I was thinking permissions to read software installers, wdyt?

Reply:
Done 👍